README.md

```diff
@@ -1,3 +1,7 @@
 # test-repo
 A simple test repository to demo cloning of git repos by 100XDevs
 Hello World
+## Development Envirnoment 
+
+Make sure you have JS available 
```

Comment:
- Ensure that you comment the necessity to include TS as well :)
- Make comments more descriptive